dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -69,4 +69,16 @@ sharedKey.desc=Specifies the string that will be used to generate the shared key
 useSystemPropertiesForHttpClientConnections=Use system properties for HTTP client connections
 useSystemPropertiesForHttpClientConnections.desc=Specifies whether to use Java system properties when the JWT Consumer creates HTTP client connections. Set this property to true if you want the connections to use the http* or javax* system properties.
 
+# Do not translate "MicroProfile", "JWT", "MP-JWT", "mp.jwt.token.header", or "MicroProfile Config"
+tokenHeader=Token header
+tokenHeader.desc=The header that is expected to contain a MicroProfile JWT. This attribute is only used by versions 1.2 and above of the MP-JWT feature. This value overrides the mp.jwt.token.header MicroProfile Config property if one is configured.
```

Comment:
Change,
"The header that is expected to contain a MicroProfile JWT. This attribute is only used by versions 1.2 and above of the MP-JWT feature. This value overrides the mp.jwt.token.header MicroProfile Config property if one is configured."
to,
"The header that is expected to contain a MicroProfile JWT. This attribute is only used by versions 1.2 and beyond of the MP-JWT feature. This value overrides the mp.jwt.token.header MicroProfile Config property if one is configured."